overrides/endless_private/search_box.js

```diff
@@ -73,7 +73,6 @@ var SearchBox = new Lang.Class({
         let cells = this._auto_complete.get_cells();
         cells[0].xpad = CELL_PADDING_X;
         cells[0].ypad = CELL_PADDING_Y;
-        cells[0].width_chars = this._get_title_max_chars();
```

Comment:
We should get rid of `_get_title_max_chars` definition too then, since this seems to be the only place where we use it.

Reply:
Done.